espnet2/gan_codec/shared/quantizer/modules/core_vq.py

```diff
@@ -315,24 +465,150 @@ def forward(self, x):
         loss = torch.tensor([0.0], device=device, requires_grad=self.training)
 
         if self.training:
-            commit_loss = F.mse_loss(quantize.detach(), x)
-            loss = loss + commit_loss
+            if self.commitment_weight > 0:
+                commit_loss = F.mse_loss(quantize.detach(), x)
+                loss = loss + commit_loss * self.commitment_weight
```

Comment:
We can remove this, as we already have commitment weight in the lambda_commit before